charts/testnet-manager/README.md

```diff
@@ -0,0 +1,20 @@
+# Substrate faucet helm chart
+
+The helm chart installs the [Testnet Manager](https://github.com/paritytech/testnet-manager).
+
+## Installing the chart
+
+```console
+helm repo add parity https://paritytech.github.io/helm-charts/
+helm install substrate-faucet parity/substrate-faucet \
```

Comment:
this doesn't correspond to the testnet-manager

Reply:
Thanks for pointing this out.